README.md

```diff
@@ -15,7 +15,7 @@ LinuxKit, a toolkit for building custom minimal, immutable Linux distributions.
 - Designed to be managed by external tooling, such as [Infrakit](https://github.com/docker/infrakit) or similar tools
 - Includes a set of longer-term collaborative projects in various stages of development to innovate on kernel and userspace changes, particularly around security
 
-LinuxKit currently supports the `x86_64`, `arm64`, and `s390x` architectures on a variety of platforms, both as virtual machines and baremetal (see [below](#booting-and-testing) for details.
+LinuxKit currently supports the `x86_64`, `arm64`, and `s390x` architectures on a variety of platforms, both as virtual machines and baremetal (see [below](#booting-and-testing) for details.)
```

Comment:
Nice catch, but can you move the `)` inside the period `.`? It applies to the entire sentence, so it should be `for details).`

Reply:
Thanks. 

1. It needs to be signed `git commit -s ...` . If you already pushed, just `git commit --amend --no-edit -s` and then `git push ... -f`
2. Please squash the two commits together so it is a single commit.

Thanks

Reply:
@deitch Yes done. and I squashed too.